migrations/__utils/util.ts

```diff
@@ -377,3 +378,25 @@ export const createReplyCommentTable = async (
     `ALTER TABLE \`${replyCommentTableName}\` COMMENT = "${postDescription} 게시글 대댓글"`,
   );
 };
+
+export const generateFkColumnAndOption = (
+  referencedTableName: string,
+  comment?: string,
+): { column: TableColumnOptions; fk: TableForeignKeyOptions } => {
```

Comment:
```suggestion
export const generateFkColumn = (
  referencedTableName: string,
  comment: string,
): { column: TableColumnOptions; fk: TableForeignKeyOptions } => {
```
1. 함수명에 option 은 불필요할거같습니다.
2. comment는 필수파라미터면 좋을거같습니다.